Src/BootCamp.Chapter/Schools/HighSchool.cs

```diff
@@ -0,0 +1,12 @@
+﻿using BootCamp.Chapter.Students;
+using System;
+using System.Collections.Generic;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5789960)